### PR TITLE
Ahmet - Added Week 1 Exercises and Practises

### PR DESCRIPTION
**Pull request by Ahmet-K**
I added Week 1 exercises (C-G)

### CODE REVIEW

**Comment by mohbeen29** [comment]
your coding is very neat and tidy! nice to look at very clear.

File: week-1-practise/level-3/1-piping.js [hunk 1]
```diff
@@ -11,6 +11,8 @@
 */
 
 var startingValue = 5;
+var result = format(add((multiply(startingValue,2)),10));
```
Comment (nbogie) [inline week-1-practise/level-3/1-piping.js]: Great!  
No need for the extra parentheses around the call to multiply, however. and I would leave space between each argument passed to each function, so:
`format(add(multiply(startingValue, 2), 10))`
But what you have is also correct.

File: week-1/D-variables/exercise.js [hunk 2]
```diff
@@ -1,3 +1,7 @@
 // Start by creating a variable `greeting`
-
+var greeting = "Hello world";
+var name = "Ahmet";
+console.log("My name is" , name , "!!!");
+console.log("My name is" + name + "!!!");
 console.log(greeting);
+console.log(`my name is ${name}!!!`)
```
[the remark this answers is not in the document]
Comment (Ahmet-K) [reply week-1/D-variables/exercise.js]: I have already fixed it.

File: week-1-practise/level-1/1-syntax-error-1.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 // There is a syntax error in this code. Fix it.
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
```
[the remark this answers is not in the document]
Comment (Ahmet-K) [reply week-1-practise/level-1/1-syntax-error-1.js]: Thanks. I got it.

File: week-1-practise/level-1/2-syntax-error-2.js [hunk 4]
```diff
@@ -1,8 +1,8 @@
 // There is a syntax error in this code. Fix it.
 
-function addNumbers(a, b, c)
+function addNumbers(a, b, c){
 return a + b + c;
-
+}
```
[the remark this answers is not in the document]
Comment (Ahmet-K) [reply week-1-practise/level-1/2-syntax-error-2.js]: Thanks. I have fixed it.

File: week-1-practise/level-1/3-syntax-error-3.js [hunk 5]
```diff
@@ -1,6 +1,6 @@
 // There is a syntax error with this code. Fix it.
 
-function(a, b, c) {
+function addNumbers(a, b, c) {
```
[the remark this answers is not in the document]
Comment (Ahmet-K) [reply week-1-practise/level-1/3-syntax-error-3.js]: Thanks.

File: week-1-practise/level-1/8-function-output-1.js [hunk 6]
```diff
@@ -6,7 +6,7 @@ var result = getNumber();
 console.log(result);
 
 // Answer these questions:
-// - What does this program do?
-// - How many parameters does the function take?
-// - What is the function name?
-// - Where is the function called? with what parameters?
\ No newline at end of file
+// - What does this program do? Create random number between 0 and 1 and then multiply to 10
```
[the remark this answers is not in the document]
Comment (Ahmet-K) [reply week-1-practise/level-1/8-function-output-1.js]: Thanks. I have fixed it.

File: week-1-practise/level-1/8-function-output-1.js [hunk 7]
```diff
@@ -6,7 +6,7 @@ var result = getNumber();
...
+// - How many parameters does the function take? -
```
[the remark this answers is not in the document]
Comment (Ahmet-K) [reply week-1-practise/level-1/8-function-output-1.js]: Thanks. I have fixed it.

File: week-1-practise/level-1/8-function-output-1.js [hunk 8]
```diff
@@ -6,7 +6,7 @@ var result = getNumber();
...
+// - What is the function name? getNumber
+// - Where is the function called? with what parameters? getNumber, no parameter
```
[the remark this answers is not in the document]
Comment (Ahmet-K) [reply week-1-practise/level-1/8-function-output-1.js]: Thanks. I got it and have fixed it.

File: week-1-practise/level-1/9-function-output-2.js [hunk 9]
```diff
@@ -6,9 +6,9 @@ var result = s('code', 'your');
 console.log(result);
 
 // Answer these questions:
-// - What does this program do? Run it and see, or Google some of the methods used ot understand them.
-// - How many parameters does the function take?
-// - What is the function name?
-// - Where is the function called? with what parameters?
-// - What does the function return?
+// - What does this program do? Run it and see, or Google some of the methods used ot understand them. The concat() method was used to join two arrays.
```
[the remark this answers is not in the document]
Comment (Ahmet-K) [reply week-1-practise/level-1/9-function-output-2.js]: Thanks. I got it and have fixed it.

File: week-1-practise/level-1/9-function-output-2.js [hunk 10]
```diff
@@ -6,9 +6,9 @@ var result = s('code', 'your');
...
+// - How many parameters does the function take? 2
```
[the remark this answers is not in the document]
Comment (Ahmet-K) [reply week-1-practise/level-1/9-function-output-2.js]: Thanks

File: week-1-practise/level-1/9-function-output-2.js [hunk 11]
```diff
@@ -6,9 +6,9 @@ var result = s('code', 'your');
...
+// - What is the function name? s
```
[the remark this answers is not in the document]
Comment (Ahmet-K) [reply week-1-practise/level-1/9-function-output-2.js]: Thanks

File: week-1-practise/level-2/1-tax.js [hunk 12]
```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(price) {
```
[the remark this answers is not in the document]
Comment (Ahmet-K) [reply week-1-practise/level-2/1-tax.js]: Thanks

File: week-1-practise/level-2/2-format-currency.js [hunk 13]
```diff
@@ -6,7 +6,9 @@
   Write a function that transforms numbers into the format £0.00
 */
 
-function formatCurrency() {}
+function formatCurrency(price) {
```
[the remark this answers is not in the document]
Comment (Ahmet-K) [reply week-1-practise/level-2/2-format-currency.js]: Thanks

File: week-1/I-floats/exercise.js [hunk 14]
```diff
@@ -1,2 +1,8 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var percentagestudent = Math.round(numberOfStudents * 100 / (numberOfStudents + numberOfMentors));
```
[the remark this answers is not in the document]
Comment (Ahmet-K) [reply week-1/I-floats/exercise.js]: Thanks.

File: week-1-practise/level-2/3-currency-conversion.js [hunk 15]
```diff
@@ -5,7 +5,9 @@
   Write a function that converts a price to USD (exchange rate is 1.4 $ to £)
 */
 
-function convertToUSD() {}
+function convertToUSD(price) {
```
[the remark this answers is not in the document]
Comment (Ahmet-K) [reply week-1-practise/level-2/3-currency-conversion.js]: Thanks. I got it and have fixed it.

File: week-1-practise/level-2/4-currency-conversion-2.js [hunk 16]
```diff
@@ -7,8 +7,16 @@
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToUSD(price) {}
-function convertToBRL(price) {}
+function convertToUSD(price) {
```
[the remark this answers is not in the document]
Comment (Ahmet-K) [reply week-1-practise/level-2/4-currency-conversion-2.js]: Thanks. I got it and have fixed it.

File: week-1-practise/level-3/1-piping.js (hunk 1 above)
Comment (Ahmet-K) [reply week-1-practise/level-3/1-piping.js]: Thanks. So, I have fixed it.

File: week-1/E-strings/exercise.js [hunk 17]
```diff
@@ -1,3 +1,16 @@
 // Start by creating a variable `message`
 
+var message = 10;
```
[the remark this answers is not in the document]
Comment (Ahmet-K) [reply week-1/E-strings/exercise.js]: Thanks

**Pull request closed without merging**